ca/certificates.go

```diff
@@ -209,23 +200,56 @@ func (rca *RootCA) RequestAndSaveNewCertificates(ctx context.Context, paths Cert
 		return nil, err
 	}
 
-	// Ensure directory exists
-	err = os.MkdirAll(filepath.Dir(paths.Cert), 0755)
+	var kekUpdate *KEKUpdate
+	for i := 0; i < 5; i++ {
+		kekUpdate, err = rca.getKEKUpdate(ctx, X509Cert, tlsKeyPair, r)
```

Comment:
This is similar to the loop for making the CSR request.  This particular function will give up if it fails.  The continuous retry occurs in `RenewTLSConfig`, which monitors the expiry and will keep retrying if it's getting close to the expiry of the TLS key.

You do bring up a good point in that we don't keep retrying if we go from an unlocked cluster to a locked cluster, though.  Maybe that should be best effort only?